episodes/excthreesteps.Rmd

```diff
@@ -0,0 +1,517 @@
+---
+title: 'Demons Image Registration'
+teaching: 10
+exercises: 2
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- How to understand and visualise three image in one pane for demons image registration algorithm?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Understanding Demons Image Registration code for 3 images viewing pane
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+# Demons Image Registration Algorithm with Multi-Pane Display
+
+## Activate Conda environment
+Open terminal, activate `mirVE` environment and run python
+
+```bash
+conda activate mirVE
+python
+```
+
+This document explains the implementation of a 2D image registration algorithm using the Demons algorithm. The implementation includes visualisation updates within a single window containing three panes for easier comparison of source, target, and warped images.
+
+## 1. Importing Libraries
+
+First, we import the necessary libraries for image processing, transformation, and visualisation. The `utils3` module is another Python file containing utility functions used for tasks such as displaying images and handling deformed and updated fields. Both `demonsReg.py` and `utils3.py` are available in `src/mirc/utils`

```

Comment:
Can you add links for files and paths (e.g., [`utils3.py`](https://github.com/HealthBioscienceIDEAS/Medical-Image-Registration-Short-Course/tree/main/src/mirc/utils3.py), etc) ? Links might not exists now but will be live when we merge to main!

Reply:
I already stated that above:
Both `demonsReg.py` and `utils3.py` are available in `src/mirc/utils`